index.html

```diff
@@ -1,50 +0,0 @@
-<html>
-
-<head>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script>
-    <script src="js/lib/jquery.ui.touch-punch.min.js"></script>
-    <link rel="stylesheet" type="text/css" href="css/styles.css">
```

Comment:
Why are we adding css in between js? It should be added first, then the js files